spec/features/visitors/email_confirmation_spec.rb

```diff
@@ -55,7 +55,12 @@
 
   context 'confirmed user is signed in and tries to confirm again' do
     it 'redirects the user to the profile' do
-      sign_up_and_2fa
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
In general I support DRY code. Seems like an anti-pattern when all the rest of this code base uses these kind of spec helper methods to authenticate in various ways. I don't feel strongly, but I'd like to hear from @monfresh and @zachmargolis on this.